### PR TITLE
raidboss: P11S Lightstream safespot

### PR DESCRIPTION
**Pull request by xiashtra**
Minimally tested.

### CODE REVIEW

**Comment by quisquous** [comment]
This didn't seem to call out anything for me across a couple of runs.  I haven't debugged any further.

**Comment by xiashtra** [comment]
The headmarkers on the cylinders are randomized, this one needs a little more work.

**Comment by xiashtra** [comment]
This should be fixed now.

File: ui/raidboss/data/06-ew/raid/p11s.ts [hunk 1]
```diff
@@ -2,13 +2,36 @@ import Outputs from '../../../../../resources/outputs';
...
+
+const headmarkers = {
+  // P11N uses, and the P11S VFX looks like, 0x00DA for Dike, but using that value gives negative results for the Arcane Cylinder headmarkers
+  // 0x01DB derived by using the P11N Styx 0x0131 headmarker and calculating backwards for Dike
+  // 0x01DB is used in DRS Trinity Seeker for Earthshaker markers, which look and behave nothing like Dike
+  dike: '01DB', // tankbuster
+  styx: '0131', // multi-hit stack, currently unused
+  orangeCW: '009C', // orange clockwise rotation
+  blueCCW: '009D', // blue counterclockwise rotation
+} as const;
```
Comment (quisquous) [inline ui/raidboss/data/06-ew/raid/p11s.ts:25]:
```suggestion
const headmarkers = {
  // vfx/lockon/eff/tank_lockon04_7sk1.avfx
  dike: '01DB', // tankbuster
  // vfx/lockon/eff/com_share4a1.avfx
  styx: '0131', // multi-hit stack, currently unused
  // vfx/lockon/eff/m0515_turning_right01c.avfx
  orangeCW: '009C', // orange clockwise rotation
  // vfx/lockon/eff/m0515_turning_left01c.avfx
  blueCCW: '009D', // blue counterclockwise rotation
} as const;
```

`01DB` in DRS is a tether value and not a headmarker value, for what it's worth.  Your values all look correct to me.  Here's some comments with the game path to the LockOn vfx.

**Comment by Wessie** [comment]
I've had to start using this recently due to colourblindness and it seems to break depending on where in the arena you are located? The headmarker events seem to come in based on distance to the orbs

**Comment by quisquous** [comment]
Do you have a network log file / timestamp for where this didn't work?

**Comment by Wessie** [comment]
Trimmed the log file to just the single encounter, it calls out `northwest` for the lightstream but it was actually `east`
[themis-trimmed.txt](https://github.com/quisquous/cactbot/files/11783289/themis-trimmed.txt)

**Comment by quisquous** [comment]
(Did you split this manually? fwiw <https://quisquous.github.io/cactbot/util/logtools/splitter.html> has a splitter and anonymizer)

**Pull request merged**